src/components/collabs/pick.svelte

```diff
@@ -27,11 +27,21 @@
 			return;
 		}
 
-		const response = _window.confirm($t('collabs.delete_pick_confirm'));
+		let reasonResponse;
 
-		if (response) {
-			await axios.delete('/api/collabs/' + collab.id + '/picks/' + pick.id);
+		if (pick.userId !== $discord?.id) {
+			reasonResponse = _window.prompt('Whats the reason for the deletion?', 'Duplicate Pick');
+		} else {
+			reasonResponse = _window.confirm($t('collabs.delete_pick_confirm'));
```

Comment:
Alright Sounds Good! Added it so it checks if the person is admin and also if it is not their own pick and then it will give the prompt else it will do a _window.confirm instead.